lib/web_ui/skwasm/text/paragraph_builder.cpp

```diff
@@ -47,3 +47,52 @@ SKWASM_EXPORT void paragraphBuilder_pop(ParagraphBuilder* builder) {
 SKWASM_EXPORT Paragraph* paragraphBuilder_build(ParagraphBuilder* builder) {
   return builder->Build().release();
 }
+
+SKWASM_EXPORT std::vector<SkUnicode::Position>* unicodePositionBuffer_create(
+    size_t length) {
+  return new std::vector<SkUnicode::Position>(length);
+}
+
+SKWASM_EXPORT SkUnicode::Position* unicodePositionBuffer_getDataPointer(
+    std::vector<SkUnicode::Position>* buffer) {
+  return buffer->data();
+}
+
+SKWASM_EXPORT void unicodePositionBuffer_free(
+    std::vector<SkUnicode::Position>* buffer) {
+  delete buffer;
+}
+
+SKWASM_EXPORT std::vector<SkUnicode::LineBreakBefore>* lineBreakBuffer_create(
+    size_t length) {
+  return new std::vector<SkUnicode::LineBreakBefore>(
+      length, {0, SkUnicode::LineBreakType::kSoftLineBreak});
```

Comment:
This is basically just zero-initialization of the vector. I have to provide some value here anyway in the C++ code to populate each element in the vector in the first place. I could add something that sets the first line break to `0, 0` in `_addLineBreakData`, but it's basically pointless since it's already `0, 0`.